### PR TITLE
Add conditionals

### LINKED ISSUE #7: Conditionals

### PR DESCRIPTION
**Pull request by eyermt**

### CODE REVIEW

File: CS_Ed_Week/CS_Ed_Week.md [hunk 1]
```diff
@@ -128,6 +128,93 @@ __Exercises__:
...
+### Python Comparison Operators:
+
+| Operator | Name | Example | Resulting Value |
+| :---: |---    |---    |--- |
+|==     |Equal  |       x == 10 | depends on x |
+|!=     |Not Equal      |       False != True | True |
+|>      |Greater than   |       100 > 10 | True |
+|<      |Less than      |       number1 < number2 | depends |
+|>=     |Greater than or equal to |     11 >= 11 | True |
+|<=     |Less than or equal to  |       string1.len() <= string2.len() | depends |
```
Comment (cuppajoe123) [inline CS_Ed_Week/CS_Ed_Week.md]: the len() function has the format `len(string1)`

File: CS_Ed_Week/CS_Ed_Week.md [hunk 2]
```diff
@@ -128,6 +128,93 @@ __Exercises__:
...
+
+#### Hypothetical train code:
+
+    if (lever == 1)
+      destination = 'Cincinatti'
+    elif (lever == 0)
+      destination = 'New York'
+    else
+      #Error: lever is broken
+      Print('STOP THE TRAIN!!!')
```
Comment (cuppajoe123) [inline CS_Ed_Week/CS_Ed_Week.md]: print() is lower case

**Comment by eyermt** [comment]
Alrighty, I fixed those errors. My lack of Python exp is shining through. Thank you for the review @cuppajoe123, you are an incredible maintainer. Keep it up!

File: CS_Ed_Week/CS_Ed_Week.md [hunk 3]
```diff
@@ -128,6 +128,92 @@ __Exercises__:
...
+The `elif` keyword is an if + else statement that can be chained to add options.
+
+    if ()
+      ...
+    elif ()
+      ...
+    elif ()
+      ...
+    else
+      ...
```
Comment (EmeraldBlock) [inline CS_Ed_Week/CS_Ed_Week.md:215]: pyhon `if` statements require a colon but do not require parentheses, such as in
```python
if lever == 1:
    # stuff
```
also, syntax higlighting might be nice

**Pull request merged**